File: 351001/Orlovich/laba_2/Domain/Models/Author.cs
namespace Domain.Models;

// Author
public class Author : IIdEntity
{
    public long id { get; set; } = 0;
    public string login { get; set; }
    public string password { get; set; }
    public string firstname { get; set; }
    public string lastname { get; set; }

    public ICollection<Issue> issues { get; set; }
}

File: 351003/Zarenok/demo/src/main/java/com/example/demo/model/Author.java
package com.example.demo.model;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;

@Entity
@Getter
@Setter
@Table(name = "authors")
public class Author {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;

    @OneToMany(mappedBy = "author", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
    private List<Issue> issues = new ArrayList<>();

}
